plugins/com.google.cloud.tools.eclipse.appengine.newproject.test/src/com/google/cloud/tools/eclipse/appengine/newproject/AppEngineWizardPageTest.java

```diff
@@ -0,0 +1,115 @@
+/*
```

Comment:
This auto-package-field-setting has been copied from the current Maven-based wizard.